mate-settings-daemon/mate-settings-manager.c

```diff
@@ -396,14 +411,125 @@ mate_settings_manager_dispose (GObject *object)
         G_OBJECT_CLASS (mate_settings_manager_parent_class)->dispose (object);
 }
 
+static void
+bus_acquired_handler_cb (GDBusConnection *connection,
+                         const gchar     *name,
+                         gpointer         user_data)
+{
+        MateSettingsManager *manager;
+
+        GError *error = NULL;
+        gboolean exported;
+
+        manager = MATE_SETTINGS_MANAGER (user_data);
+
+        g_signal_connect (manager->priv->skeleton, "handle-awake", G_CALLBACK (mate_settings_manager_awake_handler), manager);
+        g_signal_connect (manager->priv->skeleton, "handle-start", G_CALLBACK (mate_settings_manager_start_handler), manager);
+
+        exported = g_dbus_interface_skeleton_export (G_DBUS_INTERFACE_SKELETON (manager->priv->skeleton),
+                                                     connection, MSD_MANAGER_DBUS_PATH, &error);
```

Comment:
Readability: the line length is too wide making it difficult to read.

Reply:
It’s an improvement, but to make it even easier to read, use an argument on each line.